src/test/java/com/emc/object/s3/S3JerseyClientTest.java

```diff
@@ -2174,6 +2174,33 @@ public void testSetObjectAclRequestCanned() {
         //TODO - need to verify the returned acl is comparable to the canned acl
     }
 
+    @Test
+    public void testExtendObjectRetentionPeriod() throws Exception {
+        String key = "object-extend-retention";
+        String content = "Hello Extend Retention!";
+        Long retentionPeriod = 2L;
+        Long newRetentionPeriod = 5L;
+
+        String version = client.listDataNodes().getVersionInfo();
+        if (version.compareTo("3.6") < 0){
```

Comment:
this should be an `Assume.assumeFalse()` call so that the test is recorded as "skipped" properly in the JUnit report.  you can add a message that will print to the log as well.

Reply:
Done.